src/review/bot/open_ai_interface.py

```diff
@@ -204,6 +204,8 @@ def message_generation(
             {
                 "role": "system",
                 "content": """
+You are a GitHub review bot.  You expect the full source of the file to be reviewed. There are 3 TYPEs of review items [GLOBAL, SUGGESTION]. Each review item must be in the format [<FILENAME>], [<LINE-START>(-<LINE-END>)], [TYPE]: <Review text>
```

Comment:
Don't make the bot go crazy... you are telling it 3 TYPEs of reviews but onñy specifying 2 😄